src/runmode-af-packet.c

```diff
@@ -65,6 +67,49 @@ const char *RunModeAFPGetDefaultMode(void)
     return default_mode_workers;
 }
 
+/**
+ * \brief Get the af-packet configuration node for a device.
+ *
+ * Basically hunts through the list of maps for the first one with a
+ * key of "interface", and a value of the provided interface.
+ */
+static ConfNode *AFPFindDeviceConfig(ConfNode *root, const char *iface)
+{
+    ConfNode *if_root, *item;
+    const char key[] = "interface";
+
+    ConfNode *interfaces = ConfNodeLookupChild(root, "interfaces");
+    if (interfaces != NULL) {
+        return ConfNodeLookupChild(interfaces, iface);
+    }
+
+    SCLogWarning(SC_WARN_DEPRECATED,
+        "Found deprecated af-packet interface configuration. "
+        "Please switch to map based interface configuration.");
```

Comment:
I think it would be good to reference to a document/link here, as this warning is too technical to make sense to most people.